about.html

```diff
@@ -0,0 +1,172 @@
+            projects that advance the cause of the Commons, free culture and
+            open knowledge. <br />
+            So if you're active and engaged in the worlds of open content and
+            knowledge - free software advocates, wikipedians, Open knowledge,
+            galleries, libraries, museums, archives, governments and
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together
+          </p>
+          <p class="about-contact">
+            Please contact us per Email for nay further questions about CC
+            Global Summit 2015!
+          </p>
+          <p class="about-email">
+            uld@dmgmedia.com
+            <i class="fa-solid fa-regular fa-chevron-right"></i>
+          </p>
+        </div>
+      </div>
+    </section>
+    <section class="about-2" id="about-2">
+      <h2>Urban Landscape and Design Expo 2023 logo</h2>
+      <hr class="about-hr" />
+      <p class="about-2-p-1">
+        The logo of Urban Landscpae and Design 2023 was decided through the logo
+        competition from 8, June to 7, July.
+      </p>
+      <p class="about-2-p-2">
+        The logo of CC Global Summit 2015 was decided through the logo
+        competition from 8, June to 7, July. After online public voting and
+        Selection Committee’s judging process, the logo from Naresh Agrawal won
+        the prize. After consultation with Mr. Agrawal, we optimized the
+        prize-winning-logo for this year’s summit as follows:
+      </p>
+      <img src="./assests/udle-2022-main.webp" alt="desk-about" />
+      <hr class="logo-hr" />
+      <div class="past-events">
+        <h2>See the past Urban Landscape and Design Expo</h2>
+        <hr class="about-hr" />
+        <p>
+          Take a look at the past Urban Landscape and Desing Expo which took
+          place in Dubai, United Arab Emirates.
+        </p>
+      </div>
+    </section>
+    <div class="cards-section">
+      <div class="past-card past-image-1">
+        <h1>2021</h1>
+        <p>Urban Landscape and Desgin Expo in Dubai</p>
+      </div>
+      <div class="past-card past-image-2">
+        <h1>2022</h1>
+        <p>Urban Landscape and Desgin Expo in Dubai</p>
+      </div>
+    </div>
+    <!-- about section starts here -->
+
+    <!-- partners section starts here  -->
+    <section class="partner" id="partner">
+      <h1>Partners</h1>
+      <hr class="about-hr" />
+      <div class="partner-images">
+        <img src="./assests/mozilla.png" alt="mozilla" />
+        <img src="./assests/google.png" alt="google" />
+        <img src="./assests/naver.png" alt="naver" />
+        <img src="./assests/daumkakao.png" alt="daumkakao" />
+        <img src="./assests/airbnb.png" alt="airbnb" />
+      </div>
+    </section>
+    <!-- partners section ends here  -->
+
+    <!-- footer starts here  -->
```

Comment:
- [ ] Kindly note that the footer of the desktop about page has a dark background and does not have a partner section. please fix that
yours
![Screenshot (1199)](https://github.com/Kaiserabbas/Capstone/assets/65298048/e07418bc-83be-4969-9d98-b2dc8d934e8c)
original
![Screenshot (1159)](https://github.com/Kaiserabbas/Capstone/assets/65298048/a9ced63e-3eca-40e4-9432-31e7c997e8e4)